### PR TITLE
Adding issue template - ✨ New webpage

### LINKED ISSUE #806: ✨ Issue template - New webpage
@bradystroud @camillars @tiagov8

**Is your feature request related to a problem? Please describe.**
As per my conversation with Camilla and Brady, we do not have a standard for optimizing SEO when requesting/creating new webpages.

**Describe the solution you'd like**
A new Issue template in this repo (SSW.Website) that gives a starting point for anybody requesting a new webpage (this is the repo in which this is done already, but there is currently no specific Issue template for a new page). This template will make sure that each new webpage that is created has a strong SEO baseline from now on.

**Describe alternatives you've considered**
Creating a rule or SL item with instructions/checklist.
Rule - the information would be in 2 places - better to keep it in one place where all the other website changes/feedback are.
SL item - not everybody needs to create webpages, so it's unnecessary noise to make each person do it.

**Additional context**
Here is what the Issue template should look like:

**New webpage template**

Replace all the placeholders with information about the new page:

Template: if the page should follow an existing template, put it here (if not, write N/A). If you don't know where the template is but you have an example you'd like to follow (e.g. another Consulting page), put the link here.
{{ PAGE TEMPLATE }} 

Page Title: Must be descriptive and include primary keywords (Google-able). It should be no more than 60 characters.
{{ PAGE TITLE }} 

Meta Description: This is the preview that users see on Google (or other search engines). It must be a unique and clear summary of the page. It should also incorporate the primary keyword and other secondary keywords. It should be under 150 characters. If you have trouble making this, give ChatGPT your content and ask it to write one for you!
{{ META DESCRIPTION }} 

URI: Must be concise, clear, and should include the primary keywords. Use kebab case (e.g. use-dashes-and-lowercase)
{{ URI }} 

Internal Links (If applicable, include related links on the page): 
{{ RELATED URLS }} 

Content guidelines: 

- Headings – include an HTML title tag for each heading, with H1 for the main title H2 for subheadings, H3 for smaller subheadings 
   - Only one H1 per page 
   - The H1 should summarize the main content 
   - Each heading must include keywords
   - Example: 
 	        H1 – SSW Educational 
 	        H2 - How SSW can help: Customized technology solutions

- Images – must include both alt text and descriptive captions
    - Alt text describes the image to search engines 
    - Example: 
		<cat-flying.jpg> 
                Alt text - cat flying above city 
                Caption - Figure: A cat soars majestically above New York City.
	  
 - Content - where possible, add links and/or hyperlinks to other relevant pages on the website
     - Called 'internal linking', this gives pages credibility in the eyes of search engines
     - Example:
                   We are experts in Power BI. Our developers are also experienced in [database development](https://www.ssw.com.au/consulting/database-development).

Write/paste the content here, following the above guidelines for headings and images:

{{ CONTENT }}
```[tasklist]
### Tasks
- [x] Create issue template
- [x] Add PR checkbox
- [x] Update rule (include 'only H1 per page') https://www.ssw.com.au/rules/use-heading-tags-h1-h2-h3/
```

### PR DESCRIPTION
**Pull request by sethdaily**
Fixes #806 

Affected routes:
.github/ISSUE_TEMPLATE

Screenshots of the new issue form, rendered:

image
image

Screenshot of a created issue:

image

### CODE REVIEW

**Review by bradystroud — APPROVED** [review]

**Comment by wicksipedia** [comment]
screenshot tip: https://www.microsoft.com/en-us/edge/features/web-capture?form=MT00D8

File: .github/ISSUE_TEMPLATE/new_webpage.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+name: ✨ New Webpage
+description: Start here to request a new page
+title: "✨ {{ TITLE }} "
```
Comment (wicksipedia) [inline .github/ISSUE_TEMPLATE/new_webpage.yml:3]: Should we use the page emoji instead? 📄

Sparkle is meant to be used for features (code)

Comment (wicksipedia) [inline .github/ISSUE_TEMPLATE/new_webpage.yml]:
```suggestion
title: "📄 {{ TITLE }} "
```
Should we use the page emoji instead? 📄

Sparkle is meant to be used for features (code)

**Comment by wicksipedia** [comment]
@sethdailyssw can you add a screenshot of what the created issue looks like?

**Review by wicksipedia — APPROVED** [review]
lgtm

**Pull request merged**